lib/fs.js

```diff
@@ -6,6 +6,7 @@
 const constants = process.binding('constants').fs;
 const util = require('util');
 const pathModule = require('path');
+const internalFs = require('internal/fs');
```

Comment:
The diff in this file could be a lot smaller if you add `const nullCheck = internalFs.nullCheck;` up here.